libafl/src/bolts/os/windows_exceptions.rs

```diff
@@ -376,8 +381,19 @@ pub unsafe fn setup_exception_handler<T: 'static + Handler>(handler: &mut T) ->
     // SetUnhandledFilter does not work with frida since the stack is changed and exception handler is lost with Stalker enabled.
     // See https://github.com/AFLplusplus/LibAFL/pull/403
     AddVectoredExceptionHandler(
-        1,
+        0,
         Some(core::mem::transmute(handle_exception as *const c_void)),
     );
     Ok(())
 }
+
+pub type CB = unsafe extern "C" fn() -> ();
+
+extern "C" {
+    fn __sanitizer_set_death_callback(cb: CB);
+}
+
+/// Rust interface to call asan death callback
+pub unsafe extern "C" fn libafl_sanitizer_set_death_callback(cb: CB) {
+    __sanitizer_set_death_callback(cb);
+}
```

Comment:
I guess we somehow need to make this optional/depend on ASAN somehow. Else builds without linking ASAN will probably fail.

Reply:
we can just use the normal infrastructure to add a weak symbol, I assume, right?

Reply:
yup it is failing. as you can see in the gh actions result of the previous commit.
standalone libafl will, because it tries to find the symbol in sancov.lib, which is not always present there

Reply:
we can weakly define the c function and the rust api in libafl_targets.
but then the problem is when we should call this?
(because libafl does not depends on libafl_targets, we can't call this in inprocessexecutor::new() anymore)

we can mark this with __attribute__(constructor) but that'll make every windows fuzzer run this __sanitizer_set_death_callback function. (probably that does no harm, but some fuzzer like frida or tinyinst ones really don't need it)
or we can just expose the api and let the user call it in their fuzzer, but I think it's ugly

what do you think??

Reply:
If I would compile LibAFL to a `staticlib` and link against that, it might work using the code above.

Reply:
> Maybe that is actually the way to go
Yes using constructor is not possible because the callback needs to infer types. Therefore, we'll be able to call it only after we have `executor`, `event manager`, and `fuzzer`